cmd/cl-controlplane/app/server.go

```diff
@@ -129,21 +129,21 @@ func (o *Options) Run() error {
 
 	storeManager := kv.NewManager(kvStore)
 
-	// initiailze platform
-	var platform platform.Platform
+	// initialize platform
+	var pp platform.Platform
 	switch o.Platform {
 	case platformUnknown:
-		platform = unknown.NewPlatform()
+		pp = unknown.NewPlatform()
```

Comment:
use of `platform` as the variable name shadows the import of `platform` package.
In most cases this won't be an issue and would produce correct code. Potential ambiguity arises when both the  variable and package have the same function name (i.e., should it call exported package function or function with struct as the method receiver?)